### PR TITLE
Update eslint config

### PR DESCRIPTION
**Pull request by DvijIt**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -15,10 +17,19 @@ module.exports = {
...
   },
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx'],
+        moduleDirectory: ['node_modules', 'src/'],
+      },
+    },
+    react: {
+      version: '16.14',
```
Comment (andrii142) [inline .eslintrc.js]: why should we fix version here?

File: task/src/App.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from "react";
```
Comment (andrii142) [inline task/src/App.jsx]: must be reverted

**Review by andrii142 — CHANGES_REQUESTED** [review]

File: task/src/App.jsx (hunk 2 above)
Comment (DvijIt) [reply task/src/App.jsx]: reverted

File: .eslintrc.js (hunk 1 above)
Comment (DvijIt) [reply .eslintrc.js]: change to detect

**Pull request merged**